src/lua/sol/sol/config.hpp

```diff
@@ -49,34 +48,6 @@ the build system, or the command line options of your compiler.
 
 */
 
-#include <limits>
```

Comment:
Any reason these were removed?  This was added to keep assertions/crashing behavior in c++ consistent between release and debug builds.

Reply:
It's an autogenerated file. I didn't intentionally remove it. I'll try reverting the Sol changes and see if it still works.